code/missionui/chatbox.h

```diff
@@ -59,6 +59,9 @@ int chatbox_scroll_down();
 // clear the contents of the chatbox
 void chatbox_clear();
 
+// send a message to the chatbox
+void chatbox_send_msg(char msg[150]);
```

Comment:
As mentioned earlier, this should be a ptr to `msg`.

Reply:
char* and CHATBOX_MAX_LEN all handled, I think.